panacea/query_client.go

```diff
@@ -112,7 +113,68 @@ func NewQueryClient(ctx context.Context, config *config.Config, info TrustedBloc
 	go func() {
 		for {
 			time.Sleep(1 * time.Minute)
-			if err := refresh(ctx, lc, trustOptions.Period, &lcMutex); err != nil {
+			if err := refresh(ctx, lc, trustedPeriod, &lcMutex); err != nil {
+				log.Errorf("light client refresh error: %v", err)
+			}
+		}
+	}()
+
+	return &QueryClient{
+		RpcClient:         rpcClient,
+		LightClient:       lc,
+		interfaceRegistry: makeInterfaceRegistry(),
+		sgxLevelDB:        db,
+		mutex:             &lcMutex,
+	}, nil
+}
+
+func LoadQueryClient(ctx context.Context, config *config.Config) (*QueryClient, error) {
```

Comment:
Among these, there is duplicate code that is quite similar to NewQueryClient.
I'll see if there's a way to improve

Reply:
I made the improved code PR. #39 
If you think it has been improved, please approve it.  :)